tests/unit/show_tuner_decisions.c

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All rights reserved.
+ */
+
+#include "config.h"
+
+#include <assert.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdio.h>
+
+#include "nccl_ofi_tuner.h"
+
+static const char *algo_names[] = { "tree", "ring", "collnet_direct", "collnet_chain", "nvls", "nvlstree" };
+static const char *proto_names[] = { "ll", "ll128", "simple" };
+void dummy_logger(ncclDebugLogLevel level, unsigned long flags, const char *file, int line, const char *fmt, ...) { return; };
+
+int main(int argc, const char **argv) {
+  printf("nodes,ranks,size,algorithm,protocol\n");
+  for (size_t nodes = 1; nodes <= 512; nodes <<= 1) {
```

Comment:
Start with 4 given the 1 and 2 node cases will always return none?

Reply:
Why?  We should know if we change the none values, too.

Reply:
You're right, wasn't thinking straight.